### DISCUSSION TITLE
How do I get the Galaxy Brain badge?

### DISCUSSION THREAD

**Discussion by ertugrul37** [community/community#19532] [Other Feature Feedback, Questions, & Ideas] [answered]
![Ekran Alıntısı](https://user-images.githubusercontent.com/74158698/175880558-937e60da-a1a8-4efe-b10c-3be8b3659403.PNG)

**I tried but it didn't work. I did what is in the rules, but the badge did not come**

**Comment by melvincwng** [comment, marked as answer]
@ertugrul37, please follow my answer in #18293 for more information on how to get that badge.

Hope this helps.

**Reply by melvincwng** [reply]
@ertugrul37, also from your profile page, I do not see any discussions answered.

That could be the reason why the badge isn't appearing.

You need to have a **minimum of 2 accepted answers** before the badge appears.

**Reply by ertugrul37** [reply]
I'm just trying to learn github, I just started using it actively, so I couldn't do it.

**Reply by melvincwng** [reply]
@ertugrul37, just FYI do ping me here if you have any other enquiries on the latest achievement badges.

I will be glad to assist you.

**Reply by darkswann** [reply]
hello

**Reply by melvincwng** [reply]
@ertugrul37, no worries, that's why the GitHub Community forums are here for. 

It's a platform for everyone, be it new or experienced developers, to share their knowledge, technical and non-technical expertise, with everyone. So that essentially everyone benefits!

I've learned a lot on my own by scouring through the huge treasure troves of information in the various discussion threads :)

**Reply by melvincwng** [reply]
@darkswann, hello welcome to the GitHub Community 👋🏻

**Reply by ertugrul37** [reply]
> @darkswann, merhaba GitHub Topluluğuna hoş geldiniz 👋🏻

 Iwelcome

**Reply by ertugrul37** [reply]
> @ertugrul37, no worries, that's why the GitHub Community forums are here for.
> 
> It's a platform for everyone, be it new or experienced developers, to share their knowledge, technical and non-technical expertise, with everyone. So that essentially everyone benefits!
> 
> I've learned a lot on my own by scouring through the huge treasure troves of information in the various discussion threads :)

THANK YOU :)

**Reply by melvincwng** [reply]
@ertugrul37 & @darkswann, follow me on GitHub if you want to stay in touch. 

Otherwise, ping me here for any achievement-related questions, and I'll try to respond once I'm free.

Take care man, and have a wonderful day ahead 😃

**Reply by Dnkfff** [reply]
Good question, I guess it's pretty hard to do 😥

**Reply by ertugrul37** [reply]
@darkswann Did you make ?

**Reply by ertugrul37** [reply]
@Dnkfff  couldn't you too

**Reply by melvincwng** [reply]
@Dnkfff, take your time in exploring the various new achievements that GitHub has to offer.

Btw @Dnkfff , @ertugrul37 , @darkswann - purely just FYI:
- If you need a full list of all possible achievements that can be obtained on GitHub - head over to this [link](https://github.com/Schweinepriester/github-profile-achievements)
- It provides valuable resources and information on all possible achievements on GitHub up-to-date.

Hope this is useful for all of you ^^

**Comment by ertugrul37** [comment]
I'm trying but I need to answer discussion. I'm probably doing the answer wrong. There is one answer left but I still can't figure it out.
@Dnkfff 
@melvincwng 
@darkswann 
@bruce

**Reply by darkswann** [reply]
I couldn't either

**Reply by melvincwng** [reply]
@ertugrul37, I can see from your profile that you've already unlocked the 'Galaxy brain' achievement.

Congratulations on your new shiny badge 🥳 

If this was helpful, please consider **following me** on GitHub. Would love to connect with like-minded developers, and maybe even collaborate on open-source projects in the future :)

Take care and enjoy your new badge 😄

**Comment by ertugrul37** [comment]
@melvincwng mission completed 🥇 👍 💯

**Reply by sanila2007** [reply]
cool🔥

**Reply by Dnkfff** [reply]
Woah, so you've made it. The thing is that I have to create a discussion I guess :D

**Comment by sanila2007** [comment]
I think we can get a galaxy brain badge by answering to discussion (at least 2)

**Comment by janapc** [comment]
Thank you for your help

**Comment by github-staff** [comment]
Thanks so much for posting in our Community @ertugrul37. Because of an influx of users creating low quality, nonsensical or spam posts in the Community with the aim of earning certain Achievements, we’ve made the difficult decision to disable the ability to earn Achievements in our Community. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Let us know if you have any other questions 🚀
